src/router/index.js

```diff
@@ -0,0 +1,27 @@
+import Vue from 'vue'
+import VueRouter from 'vue-router'
+import HomeView from '../views/MoviesPageView.vue'
```

Comment:
It's a good practice to use `@/` as the start of your imports.

We probably won't do so, but very large Vue applications split their router into separate controller folders. In those cases you'll need to have multiple `../../../` parts instead of a simple `@`. You can also move this file around.

Also remove the `.vue` part.